app/lib/data.ts

```diff
@@ -1,21 +1,16 @@
 import OverlayBundleFactory from "@/app/services/OverlayBundleFactory";
 
-export const BUNDLE_LIST_URL = "https://raw.githubusercontent.com/bcgov/aries-oca-bundles/main";
-export const BUNDLE_LIST_PATH = "/ocabundleslist.json";
+export const BUNDLE_LIST_URL = "https://bcgov.github.io/aries-oca-bundles";
+export const BUNDLE_LIST_FILE = "ocabundleslist.json";
 
 export async function fetchOverlayBundleList() {
   try {
-    const response = await fetch(BUNDLE_LIST_URL + BUNDLE_LIST_PATH);
+    const response = await fetch(BUNDLE_LIST_URL + "/" + BUNDLE_LIST_FILE);
```

Comment:
Why the switch?  Just curious...

Reply:
More consistent with how I'm constructing paths in other parts of the code